src/components/interfaces/HMI_API.xml

```diff
@@ -2040,6 +2040,8 @@
    <param name="heatedMirrorsEnable" type="Boolean" mandatory="false">
        <description>value false means disabled, value true means enabled.</description>
    </param>
+   <param name="climateEnable" type="Boolean" mandatory="false" since="5.1">
```

Comment:
HMI API does not have tag 'since`

Reply:
@yang1070 Thanks. Fixed in 4d29747d10ac9d7d7d4294f7a7be4c35ee132c75